admob-latest/src/main/java/com/rakuten/tech/mobile/miniapp/ads/MiniAppH5AdsWebViewClient.kt

```diff
@@ -0,0 +1,14 @@
+package com.rakuten.tech.mobile.miniapp.ads
+
+import android.content.Context
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import com.google.android.gms.ads.h5.H5AdsWebViewClient
+
+class MiniAppH5AdsWebViewClient(private val context: Context) {
+    fun getH5AdsWebViewClient(webView: WebView, webViewClient: WebViewClient): H5AdsWebViewClient {
```

Comment:
Based on above comment, we also can rename this function with moving `context: Context` from constructor to here as following:

```suggestion
    /**
     * The function to create a H5AdsWebViewClient.
     * @param context comment here.
     * @param webView comment here.
     * @param webViewClientt comment here.
     */
    fun createH5AdsWebViewClient(
        context: Context,
        webView: WebView,
        webViewClient: WebViewClient
    ): H5AdsWebViewClient {
```